src/test/java/org/apache/commons/collections4/multimap/ArrayListValuedHashMapTest.java

```diff
@@ -16,8 +16,7 @@
  */
 package org.apache.commons.collections4.multimap;
 
-import java.util.List;
-import java.util.ListIterator;
+import java.util.*;
```

Comment:
No `*` imports please.

Reply:
OK，thank you, I will pay attention to similar problems